src/main/java/com/example/jspiderjava/ElementOuccrence.java

```diff
@@ -0,0 +1,21 @@
+package com.example.jspiderjava;
+
+public class ElementOuccrence {
+    public static void main(String[] args) {
+        int n = 10;
+        int result = ouccuranceOfelements(n);
+        System.out.println(result);
+    }
+    private static int ouccuranceOfelements(int n) {
```

Comment:
Explain by taking example in comments si that I understand what is the input and output. How that output came

Reply:
Here i took input n as 10 .so we have n*n*n here we will get 1000. Here we are counting zeros thats why will get result as 3